home/terminal.nix

```diff
@@ -0,0 +1,76 @@
+{ pkgs, ... }:
+
+# Platform-independent terminal setup
+{
+  # Nix packages to install to $HOME
+  #
+  # Search for packages here: https://search.nixos.org/packages
+  home.packages = with pkgs; [
+    # Unix tools
+    ripgrep # Better `grep`
+    fd
+    sd
+    tree
+
+    # Nix dev
+    cachix
+    nil # Nix language server
+    nix-info
+    nixpkgs-fmt
+    nixci
+
+    # Publishing
+    asciinema
+
+    # Dev
+    gh
+    just
+    lazygit # Better git UI
+    tmate
+  ];
+
+  home.shellAliases = rec {
+    e = "nvim";
+    g = "git";
+    lg = "lazygit";
+    t = "tree";
+  };
+
+  # Programs natively supported by home-manager.
+  programs = {
+    bat.enable = true;
+    autojump.enable = false;
+    # Type `z <pat>` to cd to some directory
+    zoxide.enable = true;
```

Comment:
Don't think you need `autojump` when there is `zoxide`?

Reply:
https://github.com/cameronraysmith/nix-dev-home/commit/b395c8f5e3bd0c97c1c54ccddf16ca374095c44a